pulp_container/tests/functional/api/test_sign_manifests.py

```diff
@@ -0,0 +1,88 @@
+import os
+import stat
+import pytest
+
+from pulp_smash.pulp3.bindings import monitor_task
+from pulp_smash.utils import execute_pulpcore_python, uuid4
+
+from pulpcore.client.pulp_container import RepositorySign
+
+from pulp_container.tests.functional.constants import REGISTRY_V2_REPO_PULP
+
+
+@pytest.fixture
+def distribution(registry_client, local_registry, container_distribution_api, add_to_cleanup):
+    """The fixture for a distribution that references a repository of the push type."""
+    image_path = f"{REGISTRY_V2_REPO_PULP}:manifest_a"
+    registry_client.pull(image_path)
+    local_registry.tag_and_push(image_path, "test-1:manifest_a")
+
+    distribution = container_distribution_api.list(name="test-1").results[0]
+    add_to_cleanup(container_distribution_api, distribution.pulp_href)
+
+    return distribution
+
+
+@pytest.fixture
```

Comment:
is this something that can be shared?

Reply:
In case some other tests will require it, then yes. But right now, we do not use it from anywhere else.

Reply:
Okay, will move it!